index.css

```diff
@@ -684,14 +685,22 @@ mark {
 }
 
 .button[disabled] {
-    color: var(--ui-color--mono);
-    fill: var(--ui-color--mono);
+    color: var(--ui-color--muted);
+    fill: var(--ui-color--muted);
+    background-color: var(--ui-color--default);
+    box-shadow: none;
```

Comment:
![image](https://user-images.githubusercontent.com/16660866/53587770-9627c000-3bce-11e9-8d94-40a4afaabb4d.png)

Base pure HTML button's disabled style has a shadow, but `.button` does not have a shadow, the fact that normal button's styles are identical, it looks inconsistent. but I might be wrong.

Reply:
you're right, this should be consistent. Nice catch!

Reply:
![image](https://user-images.githubusercontent.com/1133326/53652643-4df5b580-3c41-11e9-958e-43d0046e6a22.png)

I couldn't reproduce, probably you're looking at cached version, checked that box-shadow is applied to both raw tag and a class

Reply:
sorry, it's me who is lost. I was looking into another branch. Will fix shortly